routes/index.js

```diff
@@ -128,59 +128,16 @@ module.exports = function (passport) {
     });
   });
 
```

Comment:
I factored the route middleware out into named functions in order to make use of the "next" parameter and pass messages in req (see getStnMngment and postStnMngr below).